wordpress-mobile/WordPress-iOS
# Fix/16961 - Update existing events

Part of [#16961](https://github.com/wordpress-mobile/WordPress-iOS/issues/16961)

This PR updates Reader tracking events, so now they match proposal found in the spreadsheet shared on p9xfpQ-1eg-p2.

## Notes
- All reader tracking events (excluding `reader_reader_tag_followed` / `reader_reader_tag_unfollowed`) now contain: `blog_id`, `feed_id` and `follow` properties.
- `source` property will be added in a separate PR

## Review notes
- It is best to review this PR by going from commit to commit chronologically, since every commit addresses needed changes for a particular tracking event (including adding needed properties for a event, removing double event, fixing mistyped event, adding new event)

## To test
1. Trigger every event found in the events table found [here](https://github.com/wordpress-mobile/WordPress-Android/pull/14283) (with the generosity of @ParaskP7). Trigger can be found in the Trigger column.
2. Make sure tracking event log in the console includes: `blog_id`, `feed_id` and `follow` properties.

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
